widgetssdk/src/main/java/com/glia/widgets/chat/adapter/holder/CustomMessageViewHolder.java

```diff
@@ -0,0 +1,20 @@
+package com.glia.widgets.chat.adapter.holder;
+
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.glia.androidsdk.chat.ChatMessage;
+
+public abstract class CustomMessageViewHolder extends RecyclerView.ViewHolder {
```

Comment:
As I understand this is a public class that will be used by integrators. Then it needs javadocs. Can we remove the `custom` keyword from the name. I think it is confusing. Maybe it could be AbstractMessageViewHolder or WidgetsMessageViewHolder or something else?

Reply:
Renamed to `CustomCardViewHolder`. Like the feature name.